translations/de/info/task_short_description.html

```diff
@@ -0,0 +1 @@
+Ein Array mittels absoluten Beträgen sortieren.
```

Comment:
it depends, „mittels“ points to the task that you have to do something to have absolut values eventually, „nach“ means you already have the absolute values to sort with.


> Am 12.09.2016 um 21:17 schrieb DahliaSR <notifications@github.com>:
> 
> In translations/de/info/task_short_description.html <https://github.com/CheckiO-Missions/checkio-task-absolute-sorting/pull/16#discussion_r78436506>:
> 
> > @@ -0,0 +1 @@
> > +Ein Array mittels absoluten Beträgen sortieren.
> ... mittelsabsoluten Beträgen sortieren.
> 
> I feel like it should be: nach, but I'm not sure.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/CheckiO-Missions/checkio-task-absolute-sorting/pull/16/files/1c80697561194de559e951632c9ada7c5a6c5bad#r78436506>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ARBj_YzbYm27L6bbCiC39kjIGekeHbN0ks5qpaVbgaJpZM4J6FyH>.
>